content/github/site-policy/guidelines-for-legal-requests-of-user-data.md

```diff
@@ -194,7 +194,6 @@ In addition to the non-public user account information and account access logs m
   - Any security keys used for authentication or encryption
 
 - <a name="in-exigent-circumstances"></a>
```

Comment:
The fix for the extra unordered list item would be to remove the old anchor from the list, not removing the space
```suggestion
<a name="in-exigent-circumstances"></a>

```

Reply:
I could be missing something here (I'm relatively new to markdown, trying to translate from my HTML brain), but this _should_ actually be an unordered list item, and I think removing the space fixed it.  That said, if there's a more syntax correct way to do it, I defer!

Reply:
No, your fix is right 😄 
I get tripped up in this repo because their custom rendering doesn't seem to follow either CommonMark or GitHub Flavored Markdown rules 🤷